frontend-react/e2e/spec/chromium-only/public-pages-link-check.spec.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable playwright/no-conditional-in-test */
+import { expect, test } from "@playwright/test";
+import axios, { AxiosError } from "axios";
+import * as publicPagesLinkCheck from "../../pages/public-pages-link-check";
+
+// To save bandwidth, this test is within the /spec/chromium-only/ folder
+// Since we're just checking link validity. This is specified within our
+// Playwright.config here:
+// {
+//   name: "chromium-specific",
+//   use: { browserName: "chromium" },
+//   testMatch: "spec/chromium-only/*.spec.ts",
+// },
+
+test.describe("Evaluate links on public facing pages", () => {
+    let urlPaths: string[] = [];
+    const normalizeUrl = (href: string, baseUrl: string) =>
+        new URL(href, baseUrl).toString();
+
+    // Using our sitemap.xml that already has an up-to-date reference
```

Comment:
I think this can be shortened to 
`Using our sitemap.xml, we'll create a pathnames array`
In any case, we need to replace `great` with `create`